portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,6 +38,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String comment = request.getParameter("comment-input");
     commentsList.add(comment);
 
+    Entity commentEntity = new Entity("stringComment");
```

Comment:
nit - consider the constructor argument here: from a flexibility standpoint it might make  to use "Comment" as the kind in case these comments get expanded in the future to support a rating or an image or something else...